cli.js

```diff
@@ -15,10 +15,18 @@ var cli = meow([
 	'  $ speed-test',
 	'',
 	'Options',
-	'  --json     Output the result as JSON',
-	'  --bytes    Output the result in megabytes per second',
-	'  --verbose  Output more detailed information'
-]);
+	'  -j, --json     Output the result as JSON',
+	'  -B, --bytes    Output the result in megabytes per second (MBps)',
```

Comment:
I'm not really concerned about that. Shortflags doesn't have that kind of semantics. I would prefer `-b`. It's also JSON, not jSON though, regarding `-j`.